### PR TITLE
Example setup for tooling

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- #12755 introduced the need to download a file to run an example
- This means the example fails to run in CI without downloading that file

## Solution

- Add a new metadata to examples "setup" that provides setup instructions
- Replace the URL in the meshlet example to one that can actually be downloaded
- example-showcase execute the setup before running an example

### CODE REVIEW

**Review by BD103 — COMMENTED** [review]
What's the reason for storing the meshlet in a separate repository in the first place? (cc @JMS55)

Edit:

[It looks like the file may get updated frequently.](https://github.com/bevyengine/bevy/pull/12755#pullrequestreview-2013566714) Sounds good 👍

File: Cargo.toml [hunk 1]
```diff
@@ -986,6 +986,14 @@ name = "Meshlet"
 description = "Meshlet rendering for dense high-poly scenes (experimental)"
 category = "3D Rendering"
 wasm = false
+setup = [
```
Comment (BD103) [inline Cargo.toml:989]: Is there a place we can document this new options? Perhaps the `docs` folder?

Comment (alice-i-cecile) [reply Cargo.toml:989]: I think this probably belongs in the example-showcase somewhere? I'm not sure, but I would also like better docs.

Comment (mockersf) [reply Cargo.toml:989]: documented on the struct in the example showcase script for now

**Pull request merged**